### ISSUE TITLE
1.2.2.3.3. Chapter Verse Reference Re: Deity Rule 4.2.2. and Correct 4.2.3.

### ISSUE THREAD

**Issue by jttower** [gryphonshafer/Quizzing-Rule-Book#129]
According to the rule book, context for CVR is the single verse.  However, the Deity Rule creates a loophole here that I think could be fixed. 

Here is an example from this year:
In all of Romans/James, the CVRs where the question is "the gospel of whom" would be:

1:1 gospel of God
1:9 gospel of his Son
15:16 gospel of God
15:19 gospel of Christ

So to ALL of these questions, then, they could answer God, (pause) and then Christ, (pause) and then Son because they haven't given incorrect information according to the Deity Rule. 

Here's the thing... if it is a CVR, meaning that we are asking "according to THIS SPECIFIC verse, what is the specific answer", then it seems to me that if they say anything outside of that verse then they are out of the context of that verse. It does say in the rule book that context for CVR questions is the specific verse.  I know the deity rule as it applies to CVRs isn't delineated in the rule book, but then what is the point of having these questions if they are allowed to say things that are not IN the specific verse, or scroll through answers until they are called correct?  

I propose adding an amendment to the Deity Rule saying that in CVR questions, only the name used in the verse will be considered correct.

**Comment by jswingle** [comment]
I agree with this, but I think the issue goes beyond just the Deity Rule and might be better handled in 1.2.2.3.3 instead of 4.2.2.

Since the entire point of CVRs is to distinguish between different answers to the same question, the quizzer really shouldn't be allowed to give the answer from a different verse, even if the answer is almost identical and would otherwise be acceptable if it were an INT/MA question.

For a hypothetical example, let's say there are two verses in a chapter:

- One says "Jesus went to Jerusalem"
- Another says "Jesus went **up** to Jerusalem"

Even though these are almost identical and would not be a problem in other question types, I think the quizzer should be considered out of context if they say the wrong one. Currently, it's hard to rule on these situations, but I would be in favor of a rule saying something to the effect of "quizzers may not give answers to the same question from another verse" -- this would be  a tough rule to write precisely and clearly.

**Comment by jswingle** [comment]
I would anticipate an objection being "that sounds difficult and harsh" and it would be tough in certain specific instances, but it would also be true to the point of this question type and reward careful study. (And fix the kind of problem Jessica has highlighted, which allows quizzers to just sort of guess on some CVRs and get it right without knowing the verse.)

**Comment by ARMediting** [comment]
In Jeremy's example, I wouldn't consider "to Jerusalem" to be wrong if "up to Jerusalem" is needed, but it's definitely "not correct yet." No wrong information has been given, and to rule them incorrect there imposes a conditional word perfect standard on the question type. And I agree that the issue goes beyond the deity rule. 

Whether or not this is right, I feel like a quizzer saying a synonymous answer that's also in the chapter or another chapter is currently more likely to be counted wrong than a quizzer saying a synonymous answer not present in the material. Is this an outcome we want? Would we need to require a word perfect answer on references to avoid it?

Also, should this same conversation apply to CRs?

**Comment by jswingle** [comment]
> In Jeremy's example, I wouldn't consider "to Jerusalem" to be wrong if "up to Jerusalem" is needed, but it's definitely "not correct yet." No wrong information has been given, and to rule them incorrect there imposes a conditional word perfect standard on the question type. And I agree that the issue goes beyond the deity rule.

This seems fair to me, but what would you think if the answer was "to Jerusalem" and the quizzer said "_up_ to Jerusalem"? In that case, they've given "wrong" information to at least some extent, whether or not we consider it "wrong enough to be ruled incorrect" if you get what I mean.

> Whether or not this is right, I feel like a quizzer saying a synonymous answer that's also in the chapter or another chapter is currently more likely to be counted wrong than a quizzer saying a synonymous answer not present in the material. Is this an outcome we want? Would we need to require a word perfect answer on references to avoid it?
 
I agree that this would be the outcome, and I hadn't thought of that. I'm not sure it would be a bad outcome when talking about Reference questions, though. The point of these questions is to "distinguish between similar material appearing in different passages" so if they start saying verbatim material but from a different verse or chapter, then that seems to me the quizzer has made a fundamental error regarding the point of the question type. If the quizzer is saying a "synonymous answer not present in the material" then they aren't making an error on distinguishing between passages.

> Also, should this same conversation apply to CRs?

Yes, I think this goes for all reference Q's.

**Comment by kclimenhaga** [comment]
I can see the value in requiring word perfect answers for references. That being said, if the verse was "to Jerusalem" and the quizzer said "up to Jerusalem", I would not want to call them wrong immediately even if that phrase appears in another verse. That certainly wouldn't be wrong enough to call them wrong on a Q (assuming they went back and fixed it), so I wouldn't want it to make them wrong on a CVR.

**Comment by jswingle** [comment]
I think the issue is tricky because I want to base the Reference question rules around the _point_ of Reference questions - namely, to _distinguish similar material appearing in different passages._ By definition, this is hard to write concrete rules about unless we require word perfection.

I don't want a quizzer to be able to say "God" while answering a CVR with the answer as "Christ," even though in _any other question type_ this should be totally acceptable and the quizzer is allowed to keep answering until they say "Christ" or "Jesus." I think the quizzer should be considered 100% wrong if they mix up "the gospel of God" with "the gospel of Christ" because those distinctions are kind of, like, you know, _the whole point of Reference questions_...

BUT, I **DO** agree with @kclimenhaga that it's probably going too far to count the quizzer wrong for mixing up "to Jerusalem" vs. "up to Jerusalem." So I'm not sure how to write language that would achieve the balance I desire here.

**Comment by kclimenhaga** [comment]
Are there situations other than the Trinity rule where we would rule more strictly on reference questions than other questions? In general, I would be in favour of not prompting a quizzer for a question until the material quoted is word perfect, and of not calling the quizzer incorrect if they have said similar material (e.g. Peter vs. Simon Peter), but rather allowing the quizzer to correct the mistake. 

Even in terms of the Trinity rule, I would be hesitant to throw it out entirely for reference questions. I agree that "the gospel of God" and "the gospel of Christ" should not be considered interchangeable, if it is possible to write wording that would explain that, but I don't think that a quizzer should be immediately called incorrect for saying "the gospel of Jesus Christ" when the answer is "the gospel of Christ Jesus".

If someone can propose wording for this, I think we should consider it, but at present I suspect that changing the wording would only cause more problems than it solves.

**Comment by scottpeterson** [comment]
How about something like: 

"A quizzer quoting a word perfect phrase (of 2 words or more) that is the answer to the same base reference question, but from a different verse, will be ruled incorrect."

E.g. 
"The place of the man"
"The place of a man"

Q: "the place of whom?"

A quizzer mixing up "the man" and "a man" would be ruled incorrect, only on reference questions

I feel strongly about this for very competitive levels of quizzing.  And conversely I wouldn't want this level of exactness to be required at the district level.

**Comment by kclimenhaga** [comment]
I have a few concerns with this:
-Would a quizzer then not be called incorrect if the answer was only one word? E.g. "The gospel of God" vs. "The gospel of Christ"? To me, those are more different than a man vs. the man.
-Under this wording, then "up to Jerusalem" vs. "to Jerusalem" would have to be counted as incorrect. I don't agree with that.
-In general, I am opposed to rules that require a lot of searching by the table staff. Under this wording, the table staff would need to check each time the question phrase occurs in order to ascertain whether the quizzer's answer was found in a different verse.
-What happens when the two-word phrase is part of the answer, but not the full answer? Would the quizzer be counted incorrect for saying one word wrong if the answer was four or five words long?

I am fine with having a high standard of exactness for determining what constitutes a correct or incorrect answer, especially for references, but I am reluctant to use wording that would require quizzers being called incorrect for information that is not factually incorrect. For example, in the "up to Jerusalem" example, even if the text does not use the word "up", we know that Jerusalem is a higher elevation than most other parts of Israel, so they would have been going up to Jerusalem. If a quizzer misquotes the verse by adding "up", I don't think that they should be called incorrect. They should be allowed to correct the mistake, and then provide their question.

**Comment by scottpeterson** [comment]
Yep so this is the same problem as with normal context/correct/incorrect.   Do we want to be objective (1-2-3-4 word perfect phrases, etc), or subjective ("not factually incorrect"). 

What I'm trying to get at here, is not let quizzers mix up reference question passages. And currently there are ways a quizzer can mix up passages without being counted incorrect.

**Comment by jswingle** [comment]
I'm fine with counting quizzers incorrect on references even for answers which are not "factually incorrect." To me, this is a logical extension of the purpose of the question type: to "distinguish between exact duplicate words or phrases from the material."

For an INT/MA, we allow the quizzer to keep answering as long as they aren't saying incorrect things and eventually get to the right answer, because the purpose of these question types is to answer a straightforward question. Doesn't need to be word-perfect, but it has to be a "correct answer."

For Q/FTV, we require word perfection because the purpose of those types is to reward precise verbatim memorization.

For SIT, we are a bit more lenient than even INT/MA (the quizzer cannot "leave out key components of the quotation") because we are more interested in the quizzer's ability to accurately depict the situation than their ability to recite the quotation.

None of those question types has an explicitly spelled out "purpose" in the rulebook, but I think those are reasonable assumptions about what the purpose of each type is. 

However, with references, the rulebook actually DOES specify the purpose of the type. All the more reason for our rules to reflect this! If the quizzer gives an answer to the question that comes from a _different_ verse or chapter, I think they've made a fundamental error. It shouldn't matter whether the answer they gave is really close, or even synonymous, with the required answer.

**Comment by jswingle** [comment]
As a follow-up: I do think this rule would make references especially hard on newer quizzers, so I am all in favor of the idea @scottpeterson gave: a recommended alternate rule at district level which allows quizzers to give other answers so long as they aren't "incorrect answers"

As already stated, I also think the main rule should have latitude for _exceptionally similar_ phrasings such as "up to Jerusalem" vs. "Jerusalem" or "the Son" vs. "his Son", stuff like that. But answering "the gospel of whom?" with "Christ" when the answer is "God" should be an error.

**Comment by ARMediting** [comment]
I think we can somehow (_somehow_) strengthen the wording here without worrying about an alternate district rule. Districts are already good at being more lenient when a situation suggests it. In Central District, junior division quizzers don't have to give the correct reference question—though they're prompted to give one, helping them prepare for senior division. 

Unless we do move ahead with our idea of some kind of appendix of rules districts often tweak. So maybe we're getting at the same things.

**Comment by jswingle** [comment]
@ARMediting It's not been voted yet, but I think there should be alternate rules indicated in at least some places. Issue #113 concerns this.

**Comment by jttower** [comment]
First off, I apologize that I don't have a tl;dr version here, so bear with me...  

I think @jswingle brings up an excellent point in distinguishing the purposes of the question types, and maybe that information should be included in the rule book, perhaps at the beginning of the description of each question type under 1.2.2. I can see real value in adding that information, as I suspect not everyone understands the differences of the question types and WHY they exist.  Knowing what their purpose is might help in rulings as well. But technically this is a different issue for a different day, so I digress...

Back to the CVR/CR discussion, I agree with @kclimenhaga about not liking rules that require a lot of content searching by officials.  This would be one reason to require a word-perfect answer, because that takes away the subjectivity, however I also understand the argument against this (it's not a Q/F question).  And yes, it definitely goes beyond the Trinity Rule so I agree it should be under 1.2.2.3.  

I completely agree with @jswingle here:
 _If the quizzer gives an answer to the question that comes from a different verse or chapter, I think they've made a fundamental error._ 
So why can't we simply add this statement in 1.2.2.3: 
**If the quizzer gives an answer to the question that comes from a different verse or chapter they will be ruled incorrect.**  

I think this would be easy to rule on in most instances. Honestly, I don't like CVRs/CRs that have the same answer to the same question, but in two places.  Then you aren't distinguishing between anything since they are the same.  So in the hypothetical example of "to Jerusalem" and "up to Jerusalem", since they literally mean the same thing, I think that would be a bad reference question and should be thrown out (or not written to begin with).  I'm guessing the same would be true of @scottpeterson 's example of "the place of the man" vs. "the place of a man", unless there is significant contextual information that makes those two phrases distinguishable (but since this is a hypothetical, I can't think of a reason right now).  I would even go so far as to say that "the Son" and "his Son" are not distinguishable enough to be reference questions.  

So maybe some language also needs to be added to clarify this:
**When writing Reference questions, they must have a distinguishable counterpart with a different meaning in the same chapter (CVR) or another chapter (CR).**

And in the same vein, I move we change the example in 1.2.2.3.1. Unless I'm missing something, I don't personally think "King of whom" should be a valid CVR.  In all three verses of Luke 23, the answer would be "King of the Jews", which again, isn't distinguishing anything.  Now it could be a CR because in chapter 1 there is "King of Judea".  So this could potentially used as an example.

And while we are at it, I think the language in the second paragraph of 1.2.2.3.2. needs to be cleaned up/clarified.  Proposal:
**"If the quizzer needs to provide a reference question that is a CVR and their choice of interrogative word changes the question to a CVRMA, then the quizzer has given an invalid CVR question.  The validity of a question may change based on the interrogative word because of how the required answer changes."**

(I'm open to suggestions, especially on that last one since it's getting late and my brain is turning into a pumpkin.)  ;-)

**Comment by jswingle** [comment]
I also do not like Reference questions which indicate identical answers, and I like @jttower 's idea of banning them. The wording is good too, because it still allows for a Reference to be asked on a phrase that is repeated many times and usually has the same answer but occasionally has a different one. So for example, if a chapter has 10 instances of "The God of Israel" and 1 instance of "The God of Abraham," you can still ask a CVR on any of those 11 verses because the quizzer is still distinguishing material.

Here's an additional wrench to throw in things... what if we allow a CR to be asked on a chapter where every instance of a question has an identical answer? So if all throughout the chapter it says "the God of Israel," but elsewhere in a different chapter it says "the God of Abraham," we could ask "the God of whom" without specifying a verse...

**Comment by jswingle** [comment]
Also 100% in favor of indicating the purpose of each question type within the rulebook. Why are we asking this question type? How does it further our mission? What is the quizzer learning about Scripture by being required to answer this type?

**Comment by jttower** [comment]
@jswingle if I understand you correctly, I think it's fine for CR's like you mention (chapter 11 says God of Israel, chapter 13 says God of Abraham) because they are distinguishable between chapters.  So I don't think that's a wrench at all.  Unless I misunderstood you...

However, in your CVR example, do you mean you would write CVR questions for all 11 of the verses because one is different?  I think that is redundant, and would be more in favor of choosing one verse of the 10 "God of Israel" to contrast with "God of Abraham" (or at most two). I don't think there should be CVR's of 11:1, 11:13, 11:20, and 11:24 if they are all God of Israel.  That just enhances the probability of the answer being God of Israel if you put in 10 questions where that is the answer and only 1 where the answer is God of Abraham.

**Comment by jswingle** [comment]
@jttower I also agree that it isn't good question set construction to write EVERY instance of a CVR where there are many repetitious answers, but I wouldn't say we should make such questions invalid, because at least they _do_ "distinguish similar material." If there's any other answers within the chapter which are different, I think it's fine and good to write one or two questions which have the repetitious answer.

To clarify what I meant about the CRs, I'm talking about instances where there are multiple verses within a chapter which have the same exact phrase ("the God of Israel") and there are NO other answers to the question within that chapter. In that case, should the rules allow a chapter-only reference, since there is only one answer to the question within the chapter?

**Comment by jswingle** [comment]
This issue was opened and concerns the same proposal: https://github.com/gryphonshafer/Quizzing-Rule-Book/issues/161

There's only one post in the other issue, from @Jabberwocky415, so I've copy+pasted his post below:

--

I think a strong area of trouble for ruling on certain reference questions lies in determining when a quizzer can be called out of context. This comes into play especially in cases where the reference question in question has two answers which are similar; how do you decide if a quizzer has answered the question with the answer from a different context? An unspoken convention seems to be that a QM can make the decision that even if a quizzer gave a “similar” answer to the one written, if it’s too similar to a different answer to the same question they will be ruled out of context.

I propose we make this a more definitive rule.
An example of what I mean is from Acts 15, in verse 11 we read “the grace of our Lord Jesus”, and in verse 40 we see “the grace of the Lord”. If a quizzer jumps on a CVR on verse 40 and says “the grace of our Lord Jesus”, I think most people would agree they should be wrong, if only for the “Trinity rule (that feels weird)”, however I propose we do the same for the inverse scenario. If a quizzer jumps on a CVR on 11, and answers “the grace of the Lord” verbatim, they would be incorrect under this rule.
After all, isn’t the point of a reference question to allow a quizzer to show they can differentiate between two similar passages in the material?

Obviously this rule would have to supersede the normal Trinity Rule and out of context rules for reference questions specifically.
I think the wording should be something like:

> If the answer given is identical in wording to an answer to the same question in a different context, the quizzer will be ruled out of context

This may feel a bit extreme, but I think it removes a lot of trouble for officials when ruling on references.

**Comment by ARMediting** [comment]
Responding to @jswingle's suggestion of 

> Here's an additional wrench to throw in things... what if we allow a CR to be asked on a chapter where every instance of a question has an identical answer? So if all throughout the chapter it says "the God of Israel," but elsewhere in a different chapter it says "the God of Abraham," we could ask "the God of whom" without specifying a verse...

I'm intrigued by the idea, but I think we'd have to change the context rule for CRs. The current definition in 2.1.2. of "context" is "a range of verses from the question's reference." So I guess one option would be to make the question's content five verses about every appearance of the phrase, but that just sounds complicated for the answer judge. I'd rather just expand context to the whole chapter for CRs.

**Comment by jswingle** [comment]
I'd be completely cool with removing 5-verse context rules on CRs. It actually makes the most logical sense to do it that way -- if CVR context is limited to just the verse, CR context should be limited to just the chapter.

I don't see any competitive reason to _not_ do it this way. If a quizzer can pre-jump and give all possible sections in the chapter where their pre-jump could start a question, that's the kind of skill and preparation we would _want_ to reward. CRs would become marginally easier but would still be one of the toughest question types to prejump on -- only the really prepared quizzers would benefit from the expanded context. If the quizzer attempts to just speedquote through a chapter, they're not going to be able to figure out where the question is, so there's already a disincentive to quote _too_ fast.

**Comment by JoshJetto** [comment]
One of the issues that we discussed in our impromptu rules committee meeting at Internationals were concerns about the accessibility and understandability of Alliance Bible Quizzing. Removing the 5-verse context rule on CR questions adds to this problem, since quizzers would begin to jump on the chapter number and then quote off a bunch of random sections of the chapter. For the outsider watching, I think this would be a bit nonsensical. I am also opposed to removing the 5 verse context on CRs because it makes the jumping on CRs competitively irrelevant for all except for the CR specialists, and I would say that, as with removing the 5 verse context rule on INT questions, over-incentivizes the very top quizzers to the competitive detriment of the rest of the field, because it almost altogether removes the jumping risk for the very top quizzers, that they might jump too early and err. I also think that removing the 5 verse context on CRs, which will encourage the speed quoting through sections of the chapter, imposes a greater difficulty in the answer judging process, because it increases (sometimes greatly) the amount of material an answer judge has to be able to very quickly sift through in order to determine if the quizzer has given incorrect information or has quoted outside of the chapter in one of his/her answers. And unless the answer judges have the material memorized themselves, they are going to have a hard time doing that.

**Comment by jswingle** [comment]
@JoshJetto I'm not sure I follow why this would make jumping on CRs competitively irrelevant for non-specialists. Every question type (except for maybe Quotes?) already offers a tremendous advantage to specialists, and yet we still have generalist quizzers.

For example, a quizzer's accuracy on Finish questions when jumping at 2 syllables, easily goes from ~50% to ~85% just by studying a list. I don't think this change would be quite as drastic on CRs, particularly because CRs are _already_ the toughest question type to study a list for, barring INTs.

**Comment by JoshJetto** [comment]
The difference between text-based prompts (i.e. Finish questions) and reference-based prompts are vast in terms of the certainty with which a quizzer can jump quickly and be assured that they will get the answer correct. Reference-based prompts are always the same and tell the quizzer exactly where in the text they need to be. Thus, with the removal of the 5-verse context, the CR specialist who is able to jump on the chapter number without getting any of the text prompt can, without any challenge of competition from a quizzer who needs the text prompt to locate the answer, simply jump on the number at no real risk of error. The Finish specialist has to take an actual risk with their early jump point - that they might jump at a non-unique point in the text (like getting "Therefore"). Also, there is reasonable competition for them on Finish questions even from generalist quizzers who can legitimately risk jumping early and getting a text-prompt that is unique and beating out the Finish specialist for it. The same cannot be said for CRs if we remove the 5 verse context rule.

**Comment by jswingle** [comment]
@JoshJetto Even a quizzer with perfect material knowledge and extremely fast quoting ability is still going to error most of the time if they try this. The quizzer still has to give their question if they pre-jump, and being able to give the correct question requires the quizzer to stop and be prompted for their question at a point during their quoting when they can actually figure out what the question was. If they just keep speedquoting, when they eventually stop they'll have no clue what the question was.

The best shot I think a quizzer would have, would be jumping on a very short chapter (Hebrews 1, 1 Corinthians 13), and speedquoting one verse at a time, then stopping just long enough to be prompted by the QM. If there's no prompt, the quizzer keeps going. But even if executed perfectly, I don't think a quizzer could make it past about halfway through these chapters within 30 seconds. And surely there are multiple good CRs in some of these verses, so the accuracy would be lower than 50% even with perfect study and execution.

I think any quizzer who tried this strategy would stop after quickly discovering they nearly always error.

Worth pointing out though, that with extremely good preparation a quizzer _could_ probably do this on MACR questions. But this isn't a new problem. There are already a few chapters which only have 1 good MACR, or only have a few good MACRs which are all within a 5-verse context, etc.
